### PR TITLE
Feature/ngsv1 to jenkins

### PR DESCRIPTION
**Pull request by iariasleon**

### CODE REVIEW

File: test/acceptance/lettuce/integration/terrain.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def before_all():
     world.cb_config_to_start = ''
     world.cb_pid = get_cb_pid()
     world.bin_parms = None
-    drop_all_test_databases(world.config['mongo']['host'], int(world.config['mongo']['port']))
+    #drop_all_test_databases(world.config['mongo']['host'], int(world.config['mongo']['port']))
```
Comment (fgalan) [inline test/acceptance/lettuce/integration/terrain.py]: Maybe a "Fixme: these steps are comment by {some reason}" could be added above this line?

File: test/acceptance/lettuce/integration/terrain.py [hunk 2]
```diff
@@ -60,5 +60,5 @@ def after_each_scenario(scenario):
 
 @after.all
 def after_all(total):
-    drop_all_test_databases(world.config['mongo']['host'], int(world.config['mongo']['port']))
+    #drop_all_test_databases(world.config['mongo']['host'], int(world.config['mongo']['port']))
```
Comment (fgalan) [inline test/acceptance/lettuce/integration/terrain.py]: Maybe a "Fixme: these steps are comment by {some reason}" could be added above this line?

File: test/acceptance/lettuce/integration/terrain.py [hunk 3]
```diff
@@ -20,16 +20,16 @@
...
 
+
 from lettuce import before, world, after
 from integration.tools.general_utils import stop_mock, drop_all_test_databases, check_properties, get_cb_pid, stop_cb
 from iotqautils.iotqaLogger import get_logger
 
 @before.all
 def before_all():
-    world.log = get_logger('lettuce', world.config['environment']['log_level'], True, True, 'logs/lettuce.log')
+    world.log = get_logger('lettuce', world.config['environment']['log_level'], True) #, True, 'logs/lettuce.log')
```
Comment (fgalan) [inline test/acceptance/lettuce/integration/terrain.py]: Maybe a "Fixme: last parameters are commented by {some reason}" could be added above this line?

Comment (iariasleon) [inline test/acceptance/lettuce/integration/terrain.py]: Fixed in 0fbe224

File: test/acceptance/lettuce/integration/terrain.py (hunk 2 above)
Comment (iariasleon) [inline test/acceptance/lettuce/integration/terrain.py]: Fixed in 0fbe224

File: test/acceptance/lettuce/integration/terrain.py (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/lettuce/integration/terrain.py]: Fixed in 0fbe224

**Comment by fgalan** [comment]
LGTM

**Pull request merged**